PluginsAndFeatures/azure-toolkit-for-intellij/src/main/kotlin/com/microsoft/intellij/ui/JTextAreaWithTheme.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.intellij.ui
+
+import com.microsoft.azure.hdinsight.common.DarkThemeManager
+import org.apache.commons.lang3.StringUtils
+import javax.swing.JTextArea
+
+class JTextAreaWithTheme : JTextArea() {
```

Comment:
Might the name ‘ErrorTextArea’ be more sense?

Reply:
Fixed in new iteration